tests/cms/views/regions/delete_region.py

```diff
@@ -70,11 +72,15 @@ def test_delete_all_regions_is_successful(
             assert response.status_code == 302
             redirect = response.headers.get("location")
             response = client.get(redirect)
-            assert "Region wurde erfolgreich gelöscht" in response.content.decode(
-                "utf-8",
+            assert (
+                "Dieser Vorgang kann mehrere Minuten dauern."
+                in response.content.decode(
+                    "utf-8",
+                )
             )
 
     if role in [CMS_TEAM, SERVICE_TEAM, ROOT]:
+        time.sleep(10)
```

Comment:
Maybe the speed and also the robustness of the test could be improved by having a flexible wait:
```suggestion
        for n in range(0, 30):
            time.sleep(1)
            if Region.objects.count() == 0:
                break
```